### DISCUSSION TITLE
how do I make an automatic message reply in chatwoot using Whatsapp API Cloud according to the template that I have created?

### DISCUSSION THREAD

**Discussion by ahmad-septian** [chatwoot/chatwoot#8875] [Chatwoot APIs]
### Describe the bug

I have a project for booking tickets I want to make ticket reservations via whatsapp, but I'm confused about how to make it so that when a user or customer orders via whatsapp, the one who answers is a bot or automatically receives a message back, I've been looking for ways but still haven't found this solution. I'm using version 3.4.0.

### To Reproduce

I have configured the webhook in Meta to chatwoot nature.

### Environment

app.chatwoot.com

### Cloud Provider

None

### Platform

Browser

**Comment by linear** [comment]
<p><a href="https://linear.app/chatwoot/issue/PR-895/how-do-i-make-an-automatic-message-reply-in-chatwoot-using-whatsapp">PR-895 how do I make an automatic message reply in chatwoot using Whatsapp API Cloud according to the template that I have created?</a></p>

**Comment by sojan-official** [comment]
There are multiple ways 

- you can configure webhooks on Chatwoot so that Chatwoot will send a webhook notification to your external service from where you can create a response back to the contact via Chatwoot APIs
- if the message is static, you can look into automations in Chatwoot.
